conf/evolutions_postgres/010-add-organization-data.sql

```diff
@@ -0,0 +1,9 @@
+-- https://github.com/scalableminds/webknossos/pull/2594 TODO PR number
+
+-- UP:
+
+ALTER TABLE webknossos.organizations ADD COLUMN additionalInformation VARCHAR(256);
+
+-- DOWN:
+
+ALTER TABLE webknossos.organizations DROP COLUMN additionalInformation;
```

Comment:
Did you test both directions locally? We might need to drop the organizations_ view before and recreate it afterwards. (If that proves necessary, let’s also group the statements with BEGIN; and END; to make them transactional)

Reply:
Yes and it works

Reply:
did not work for me. added a commit fixing this. odd that it worked for you.